scripts/SolCx/pureshear_ALE_restart.jl

```diff
@@ -0,0 +1,215 @@
+using Statistics, LinearAlgebra, Printf, Base.Threads, CairoMakie, JLD2
+const year     = 365*3600*24
+const USE_GPU  = false
+
+using JustPIC, JustPIC._2D
+const backend = JustPIC.CPUBackend 
+
+const ALE    = true
+last_step    = 100
+restart      = false
```

Comment:
```suggestion
const last_step    = 100
const restart      = false
```

Every living in the global scope should be marked as `const`

Reply:
hmmmm really? Then it's annoying as one has to restart julia to modify them?  I will move them to the main() scope then.